client/src/context/useKanbanContext.tsx

```diff
@@ -1,45 +1,84 @@
-import { useState, useContext, createContext, FunctionComponent } from 'react';
+import { useState, useContext, createContext, FunctionComponent, useEffect } from 'react';
 import { DraggableLocation, DropResult } from 'react-beautiful-dnd';
-import { columnData } from '../components/Kanban/data';
+import cloneDeep from 'lodash.clonedeep';
+import { getUserBoards, updateBoard } from '../helpers/';
+import { useSnackBar } from './useSnackbarContext';
 import { IKanbanContext } from '../interface/KanbanContext';
 import { IColumn } from '../interface/Column';
 import { ICard } from '../interface/Card';
-import cloneDeep from 'lodash.clonedeep';
-import { useSnackBar } from './useSnackbarContext';
+import { IBoard } from '../interface/Board';
+import { useAuth } from './useAuthContext';
 
 export const KanbanContext = createContext<IKanbanContext>({} as IKanbanContext);
 
 export const KanbanProvider: FunctionComponent = ({ children }): JSX.Element => {
-  const [columns, setColumns] = useState<Array<IColumn>>(columnData);
+  const [activeBoard, setActiveBoard] = useState<IBoard>({
+    _id: 'Initial',
+    name: 'Initial',
+    columns: [],
+    cards: [],
+    user: 'Initial',
+    createdAt: 'Initial',
+  });
+  const [columns, setColumns] = useState<IColumn[]>(activeBoard.columns);
   const [focusedCard, setFocusedCard] = useState<ICard | null>(null);
   const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser } = useAuth();
+
+  useEffect(() => {
+    if (loggedInUser) getFirstBoard();
+
+    return;
+  }, [loggedInUser]);
+
+  // Helper function to set the first board;
+  const getFirstBoard = async () => {
+    const request = await getUserBoards();
+    const board = request.boards[0];
+    setActiveBoard(board);
+    setColumns(board.columns);
+
+    return board;
+  };
```

Comment:
Function to set up an initial board (the user's first board) as long as they're logged in